pico/api/src/main/java/io/helidon/pico/api/ElementInfoBlueprint.java

```diff
@@ -19,66 +19,22 @@
 import java.util.Optional;
 import java.util.Set;
 
-import io.helidon.builder.Builder;
-import io.helidon.builder.Singular;
-import io.helidon.common.types.AnnotationAndValue;
+import io.helidon.builder.api.Prototype;
+import io.helidon.common.types.Annotation;
+import io.helidon.common.types.TypeName;
+import io.helidon.config.metadata.ConfiguredOption;
 
 /**
  * Abstractly describes method or field elements of a managed service type (i.e., fields, constructors, injectable methods, etc.).
  */
-@Builder(allowPublicOptionals = true)
-public interface ElementInfo {
+@Prototype.Blueprint
```

Comment:
what became of allowPublicOptionals - is that the default behavior of the builders?

Reply:
public optionals are not allowed.